src/test/java/com/znsio/teswiz/steps/VodQASteps.java

```diff
@@ -37,4 +35,13 @@ public void scrollToElement() {
     public void isElementWithTextVisible() {
         new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).isElementWithTextVisible();
     }
+    @When("I tap in the middle of the screen")
+    public void iTapInTheMiddleOfTheScreen() {
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).tapInTheMiddleOfTheScreen();
+    }
+
+    @Then("I am able to move from {string} page to next page")
+    public void iAmAbleToMoveFromPageToNextPage(String pageHeader) {
```

Comment:
Maybe it would be better to name the variable as something else, maybe screenHeading, screenTitle.
If it seems ok according to functionality then let it be.